centaur/src/main/config/jes-compose.yml.ctmpl

```diff
@@ -15,7 +15,7 @@ cromwell:
   ports:
     - "8000:8000"
   environment:
-    JAVA_OPTS: -Dscala.concurrent.context.minThreads=50 -Dscala.concurrent.context.numThreads=50 -Dscala.concurrent.context.maxThreads=50 -Xmx20g -Xms20g -Dbackend.defaultBackend=JES -Dbackend.jes.project=broad-dsde-cromwell-dev -Dbackend.jes.baseExecutionBucket=gs://cloud-cromwell-dev/centaur-executions -Dbackend.jes.endpointUrl=https://genomics.googleapis.com/ -Ddatabase.config=main.default -Ddatabase.main.default.db.url=jdbc:mysql://mysql:3306/cromwell -Ddatabase.main.default.db.driver=com.mysql.jdbc.Driver -Ddatabase.main.default.driver=slick.driver.MySQLDriver$$ -Ddatabase.main.default.db.user=cromwell -Ddatabase.main.default.db.password=cromwell -Dgoogle.applicationName=cromwell -Dgoogle.cromwellAuthenticationScheme=service_account -Dgoogle.serviceAuth.pemFile=/etc/cromwell-account.pem -Dgoogle.serviceAuth.serviceAccountId=cromwell@broad-dsde-cromwell-dev.iam.gserviceaccount.com -Dkamon.auto-start=true
+    JAVA_OPTS: -Dscala.concurrent.context.minThreads=50 -Dscala.concurrent.context.numThreads=50 -Dscala.concurrent.context.maxThreads=50 -Xmx20g -Xms20g -Dbackend.defaultBackend=JES -Dbackend.jes.project=broad-dsde-cromwell-dev -Dbackend.jes.baseExecutionBucket=gs://cloud-cromwell-dev-self-cleaning/centaur-executions -Dbackend.jes.endpointUrl=https://genomics.googleapis.com/ -Ddatabase.config=main.default -Ddatabase.main.default.db.url=jdbc:mysql://mysql:3306/cromwell -Ddatabase.main.default.db.driver=com.mysql.jdbc.Driver -Ddatabase.main.default.driver=slick.driver.MySQLDriver$$ -Ddatabase.main.default.db.user=cromwell -Ddatabase.main.default.db.password=cromwell -Dgoogle.applicationName=cromwell -Dgoogle.cromwellAuthenticationScheme=service_account -Dgoogle.serviceAuth.pemFile=/etc/cromwell-account.pem -Dgoogle.serviceAuth.serviceAccountId=cromwell@broad-dsde-cromwell-dev.iam.gserviceaccount.com -Dkamon.auto-start=true
```

Comment:
ToL this line is getting intense.  maybe there's some docker-compose-fu that can build it from smaller pieces?

Reply:
👍 It's YAML. Unlike JSON we can make it [multiline](https://yaml-multiline.info/).